documentation/src/main/markdown/packagemgrs/python.md

```diff
@@ -99,4 +100,17 @@ Poetry detector attempts to run on your project if either a poetry.lock or pypro
 The Poetry detector parses poetry.lock for dependency information. If the detector discovers a pyproject.toml file but not a poetry.lock file, it will prompt the user to generate a poetry.lock by running `poetry install` and then run [company_name] [solution_name] again.
 The Poetry detector extracts the project's name and version from the pyproject.toml file.  If it does not find a pyproject.toml file, it will defer to values derived by git, from the project's directory, or defaults.
 
-When the `--detect.poetry.dependency.groups.excluded` property is specified, presence of both poetry.lock and pyproject.toml files is required for this detector to run successfully.
\ No newline at end of file
+When the `--detect.poetry.dependency.groups.excluded` property is specified, presence of both poetry.lock and pyproject.toml files is required for this detector to run successfully.
+
+## Setuptools detector
+
+Setuptools detector attempts to run on your project, after potential execution of other detectors, if a pyproject.toml file containing a build section with `requires = ["setuptools"]` or equivalent line is located. Setuptools scans can be run in both build, if a pip installation is available, and buildless mode if not.
+
+<note type="note">Setuptools build detector should be run in a virtual environment, or environment with a clean global pip cache, where a pip install has only been performed for the project being scanned.</note>
+
+[company_name] [solution_name] parses the pyproject.toml file to determine if the `[build-system]` section has been configured for Setuptools via the `requries= ["setuptools"]` setting. If the setting is located and pip is installed in the environment, either in the default location or specified via the `--detect.pip.path` property, the detector will execute in a virtual environment. If the detector discovers a configured pyproject.toml file but not a pip executible, it will execute in buildless mode where it will parse dependencies from the pyproject.toml, setup.cfg, or setup.py files but may not be able to specify exact package versions.
```

Comment:
The build detector will also parse the toml, cfg, and py looking for dependencies. It will then run pip show on them to get enough detail about their versions and relationships to build a BDIO. Just calling out we need dependencies in one of those three files for both build and buildless.

Reply:
@dterrysynopsys Does the detector look in all three files for the `requries= ["setuptools"]` setting prior to running, or only in the .toml file, and then it would look in all three for dependencies? (Just making sure I am accurate.)

Reply:
@sig-cpotts we will always look for requries= ["setuptools"] in the pyproject.toml. If it isn't there or there is no pyproject.toml then these detectors will not run and another Python detector will make attempts at handling the project. If we have a pyproject.toml and it has that requires line then we will look in the following order for dependencies: pyproject.toml, setup.cfg, setup.py. If there are no dependencies in any of these files then we will fail and another Python detector will run. Sounds like from your statement you have this right, sorry if I added any confusion.